Sources/OpenAI/AIProxy/AIProxyService.swift

```diff
@@ -44,8 +44,10 @@ struct AIProxyService: OpenAIService {
     self.organizationID = organizationID
     self.debugEnabled = debugEnabled
     openAIEnvironment = .init(baseURL: serviceURL ?? "https://api.aiproxy.pro", proxyPath: nil, version: "v1")
+    self.httpClient = NoOpClient()
   }
 
+  let httpClient: HTTPClient
   let session: URLSession
```

Comment:
I'm not super familiar with the code, but it seems weird to see both  HTTPClient and URLSession properties here.

Reply:
why not remove the `session` property, the `NoOpClient` type, and do
```swift
self.httpClient = URLSessionHTTPClientAdapter(urlSession: URLSession(
      configuration: .default,
      delegate: aiproxySecureDelegate,
      delegateQueue: nil))
```

Reply:
This was a good idea, removed the `NoOpClient` in favor of your solution!